lib/cli/cli.js

```diff
@@ -156,9 +158,10 @@ class CLI {
         loggerTesting.info('cli: command.validateAndRun');
 
         return command.validateAndRun(commandArgs);
-
       }).finally(() => {
-        instrumentation.stopAndReport('command', commandName, commandArgs);
+        if (initCompleted) {
+          instrumentation.stopAndReport('command', commandName, commandArgs);
+        }
```

Comment:
The only code wrapped inside a promise handler prior to the flag being set is

```js
        instrumentation.stopAndReport('init');
```

So the only way the flag is going to be unset is because we error'd when trying to invoke `stopAndReport('init')` (eg because the addon instrumentation hook errored).  What's the win in retrying `stopAndReport('init')` here?  Is there a case I've forgotten?